From d1ab2f62a03c2c977de6fed5fca8de63e328a870 Mon Sep 17 00:00:00 2001
Message-Id: <d1ab2f62a03c2c977de6fed5fca8de63e328a870.1391527986.git.Jim.Somerville@windriver.com>
From: Jim Somerville <Jim.Somerville@windriver.com>
Date: Tue, 4 Feb 2014 15:30:41 +0000
Subject: [PATCH 1/1] openvswitch: add more target python substitutions

The TARGET_PYTHON variable is used for script
substitutions to ensure the scripts will be able to properly execute
if the target system has a different path for the python.

Signed-off-by: Jim Somerville <Jim.Somerville@windriver.com>
---
 ovsdb/ovsdb-dot.in         | 2 +-
 ovsdb/ovsdb-idlc.in        | 2 +-
 utilities/ovs-dpctl-top.in | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/ovsdb/ovsdb-dot.in b/ovsdb/ovsdb-dot.in
index 85c126d..402a77c 100755
--- a/ovsdb/ovsdb-dot.in
+++ b/ovsdb/ovsdb-dot.in
@@ -1,4 +1,4 @@
-#! @PYTHON@
+#! @TARGET_PYTHON@
 
 from datetime import date
 import ovs.db.error
diff --git a/ovsdb/ovsdb-idlc.in b/ovsdb/ovsdb-idlc.in
index ec1c655..f5c135f 100755
--- a/ovsdb/ovsdb-idlc.in
+++ b/ovsdb/ovsdb-idlc.in
@@ -1,4 +1,4 @@
-#! @PYTHON@
+#! @TARGET_PYTHON@
 
 import getopt
 import os
diff --git a/utilities/ovs-dpctl-top.in b/utilities/ovs-dpctl-top.in
index f43fdeb..8475118 100755
--- a/utilities/ovs-dpctl-top.in
+++ b/utilities/ovs-dpctl-top.in
@@ -1,4 +1,4 @@
-#! @PYTHON@
+#! @TARGET_PYTHON@
 #
 # Copyright (c) 2013 Nicira, Inc.
 #
-- 
1.8.3.4

